src/odbc/unittests/stats.c

```diff
@@ -49,13 +48,25 @@ TestProc(const char *type, const char *expected)
 }
 
 static void
-TestTable(const char *type, const char *expected)
+TestTable(const char *catalog, const char *type, const char *expected)
 {
 	char sql[256];
+	char *schema = NULL;
+
+	if (catalog) {
+		schema = "dbo";
+		sprintf(sql, "DROP DATABASE IF EXISTS %s", catalog);
+		odbc_command(sql);
+		sprintf(sql, "CREATE DATABASE %s", catalog);
+		odbc_command(sql);
```

Comment:
requirements for the user are not so high to be able to create and destroy databases, see http://www.freetds.org/userguide/confirminstall.htm#TESTS, but well, nothing is set on stone. But I would instead asking to create (manually in the documentation) a database and have the user as owner of this database.

Reply:
Sure, I'll:

* Create the `testDB` DB in misc/appveyor.xml via `sqlcmd`
* Update the requirements in doc/userguide.xml
* Simplify the unit-test here knowing that the DB exists.

Reply:
Or would be possible, knowing `tempdb` and `master` are always available to save the current database name, move to either `tempdb` or `master` and use original database for tests. Not sure how easy but should be possible. Won't work on Azure as I think there you need to specify a database at login time and you can't switch but it's not a big deal. Or require to create a `freetds_test` database (or simething similar), create it on Appveyor and document on the user guide.

Reply:
The `freetds_test` DB seemed relatively simple - added a new commit.

Thanks again